From 6ac4c6f887f5a8efb6a6952798c09a2562022966 Mon Sep 17 00:00:00 2001
Message-Id: <6ac4c6f887f5a8efb6a6952798c09a2562022966.1648734629.git.geliang.tang@suse.com>
From: Wan Jiabing <wanjiabing@vivo.com>
Date: Thu, 1 Apr 2021 15:20:37 +0800
Subject: [PATCH] bpf: Remove repeated struct btf_type declaration
Git-commit: 6ac4c6f887f5a8efb6a6952798c09a2562022966
Patch-mainline: v5.13-rc1
References: bsn#127
Modified-by-SEL: No


struct btf_type is declared twice. One is declared at 35th line. The below
one is not needed, hence remove the duplicate.

Signed-off-by: Wan Jiabing <wanjiabing@vivo.com>
Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
Acked-by: Song Liu <songliubraving@fb.com>
Link: https://lore.kernel.org/bpf/20210401072037.995849-1-wanjiabing@vivo.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 include/linux/bpf.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/linux/bpf.h b/include/linux/bpf.h
index 9fdd839b418c..ff8cd68c01b3 100644
--- a/include/linux/bpf.h
+++ b/include/linux/bpf.h
@@ -928,7 +928,6 @@ struct bpf_link_primer {
 };
 
 struct bpf_struct_ops_value;
-struct btf_type;
 struct btf_member;
 
 #define BPF_STRUCT_OPS_MAX_NR_MEMBERS 64
-- 
2.34.1

